pkg/test/vpc_client/bastion.go

```diff
@@ -87,9 +88,64 @@ func (vpc *VPC) PrepareBastion(zone string) (*types.Instance, error) {
 	}
 	if len(insts) == 0 {
 		log.LogInfo("Didn't found an existing bastion, going to launch one")
-		return vpc.LaunchBastion("", zone)
+		if cidrIp == "" {
+			cidrIp = CON.RouteDestinationCidrBlock
+		}
+		_, _, err = net.ParseCIDR(cidrIp)
+		if err != nil {
+			log.LogError("CIDR IP address format is invalid")
+			return nil, err
+		}
+		userData := fmt.Sprintf("#!/bin/bash\n\t\t"+
```

Comment:
Use `\`` to remove the escape characters to make it readable.
```suggestion
		userData := fmt.Sprintf(`#!/bin/bash\n\t\t"+
```